platform/ios/src/MGLAnnotationImage.m

```diff
@@ -57,15 +57,11 @@ - (BOOL)isEqual:(id)other {
     
     return ((!_reuseIdentifier && !otherAnnotationImage.reuseIdentifier) || [_reuseIdentifier isEqualToString:otherAnnotationImage.reuseIdentifier])
     && _enabled == otherAnnotationImage.enabled
-    && ((!_image && !otherAnnotationImage.image) || [UIImagePNGRepresentation(_image) isEqualToData:UIImagePNGRepresentation(otherAnnotationImage.image)]);
```

Comment:
@frederoni, any reason why comparing two UIImages for object equality isn’t good enough?